components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/impl/UserstoresApiServiceImpl.java

```diff
@@ -41,9 +42,21 @@
     @Autowired
     private ServerUserStoreService serverUserStoreService;
 
+
+    private static boolean isAvailableUserStoreTypes(List<AvailableUserStoreClassesRes> userStoreList, String typeID) {
+        for (AvailableUserStoreClassesRes userStore : userStoreList) {
```

Comment:
add new line after method signature

Reply:
Fixed in commit f5ea11e14fef1323d232a58286f4ebb697559da1